Assets/Shopify/UIToolkit/Theme Controllers/CartController.cs

```diff
@@ -0,0 +1,122 @@
+namespace Shopify.UIToolkit {
+    using UnityEngine.Events;
+    using Shopify.Unity;
+    using Shopify.Unity.SDK;
+    using System.Linq;
+
+    public class CartController {
```

Comment:
I think you want this to be serializable as well

Reply:
Will nested classes/instances be rendered in the editor? So like the controller will render but I'm not sure if the `UnityEvents` for the cart will currently be useable.